templates/app-ts/tsconfig.json

```diff
@@ -1,22 +1,17 @@
 {
     "compilerOptions": {
-        "module": "commonjs",
+        "target": "ES2018",
+        "module": "CommonJS",
+        "moduleResolution": "node",
         "esModuleInterop": true,
         "allowSyntheticDefaultImports": true,
-        "target": "es6",
-        "moduleResolution": "node",
+        "strict": true,
         "sourceMap": true,
         "outDir": "dist",
         "baseUrl": ".",
         "paths": {
-            "*": [
-                "node_modules/*"
-            ]
+            "*": ["node_modules/*"]
```

Comment:
This is not really needed since node_modules is already included.

Reply:
I think this will be needed for `tsconfig.test.json`, I'll move it there (with a `../`).

Reply:
As far as I know, `node_modules` is always included when `moduleResolution` is set to `node`, otherwise if multiple levels of`node_modules` are provided it will not work.
More info here: https://www.typescriptlang.org/docs/handbook/module-resolution.html#node

Reply:
Oh, yeah, I forgot about walking "up" the hierarchy. Then this can just be removed.